utils/epinow2/functions.py

```diff
@@ -39,7 +63,7 @@ def generate_uuid() -> UUID:
     return uuid1()
 
 
-def generate_job_id(job_id: UUID | None = None, as_of_date: int | None = None) -> str:
+def generate_job_name(job_id: UUID | None = None, as_of_date: int | None = None) -> str:
```

Comment:
I kept the timestamp as an int because it's parsed into iso format here; that fails if the datetime is a string